src/cascadia/TerminalApp/GlobalAppSettings.h

```diff
@@ -84,8 +100,24 @@ class TerminalApp::GlobalAppSettings final
     bool _copyOnSelect;
     winrt::Windows::UI::Xaml::ElementTheme _requestedTheme;
 
+    winrt::TerminalApp::LaunchMode _launchMode;
+
     static winrt::Windows::UI::Xaml::ElementTheme _ParseTheme(const std::wstring& themeString) noexcept;
     static std::wstring_view _SerializeTheme(const winrt::Windows::UI::Xaml::ElementTheme theme) noexcept;
 
+    static void _ParseInitialPosition(const std::wstring& initialPosition,
+                                      int32_t& initialX,
+                                      bool& isInitialXSet,
+                                      int32_t& initialY,
+                                      bool& isInitialYSet) noexcept;

```

Comment:
_isInitialXSet and _isInitialYSet don't need to be tracked separately now that _initialX/Y are std::optional<>.  Can you eliminate those bools, pass the parameters as std::optional<> to these functions, and use _initialX.has_value() in any place which wants to know if it is set?

Reply:
Removed the useless bools.